### ISSUE TITLE
Deal with this 🙃

### ISSUE THREAD

**Issue by nmanaud** [openplanetary/opm#3]
<img width="350" alt="mars_basemaps" src="https://user-images.githubusercontent.com/1929139/30230350-304ce02a-94e6-11e7-9472-10ae47b5129e.png">

**Comment by aprossi** [comment]
yes, also in other cases, e.g. http://openplanetarymap.org/basemaps/#6/-21.085/-101.338 (Claritas Rupes)

**Comment by aprossi** [comment]
or.. http://openplanetarymap.org/basemaps/#7/-48.195/-153.083 (is the azimuth of the features affected somehow involved/related?)

**Comment by aprossi** [comment]
another one.. http://openplanetarymap.org/basemaps/#7/-53.752/-44.264 azimuth?

**Comment by aprossi** [comment]
and this.. http://openplanetarymap.org/basemaps/#7/-24.562/99.833

**Comment by rachjoseph3** [comment]
Here's another one: Oenotria Scopuli - [http://openplanetary.co/opm/#7/-6.026/1157.948](http://openplanetary.co/opm/#7/-6.026/1157.948)
